.pack.mk

```diff
@@ -22,17 +22,28 @@ ifeq (${VERSION}, $(filter ${VERSION}, 1.10))
 endif
 
 prepare:
-	rm -rf build packpack
-	git clone https://github.com/packpack/packpack.git
+#	rm -rf build packpack
+#	git clone https://github.com/packpack/packpack.git
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
 		if [ -n "${GIT_TAG}" ]; then \
 			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
 		else \
-			export VERSION="$$(echo ${GIT_DESCRIBE} | sed ${SED_REPLACE_VERSION_REGEX} | sed 's/-/~/').dev"; \
+			if [ "${OS}" = "alpine" ]; then \
+				if git describe | grep -q "entrypoint" ; then \
```

Comment:
We already have `GIT_DESCRIBE`, should not we use it here? By the way, it was `git describe HEAD`

Reply:
outdated

Reply:
Please use `echo ${GIT_DESCRIBE}` here like you do in all other places.

Reply:
OK